alpine-nix-rails-nginx/etc/runit/1

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source /etc/profile.d/nix.sh
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
```

Comment:
It looks like the change that causes problems was removed upstream. I'm going to remove the line that causes problems in our nix.sh using `sed`. When the next version of `nix` is released with the change I'll remove the `sed` line.